db/migrate/20220824174034_add_confirmable_to_devise.rb

```diff
@@ -0,0 +1,8 @@
+class AddConfirmableToDevise < ActiveRecord::Migration[6.1]
+  def change
+    change_table(:users) do |t| 
+      t.confirmable 
+    end
+    add_index  :users, :confirmation_token, :unique => true 
+  end
+end
```

Comment:
![image](https://user-images.githubusercontent.com/65068771/186531488-dcf9d396-a4a2-487c-9faa-6c40edbfa2ab.png)

- [ ] Kindly address the issue started by the previous reviewer, this is due to some duplicated stuff and errors in migrations. Note that this migration is useless as all the logic of confirmable is done in your `add_devise_to_user` migration. You can solve the error above by removing this migration.